<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string extracomment="application name">File Monitor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/icons/file-monitor.xpm</normaloff>:/icons/file-monitor.xpm</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="central_widget"/>
  <widget class="QStatusBar" name="status_bar">
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
  </widget>
  <widget class="QToolBar" name="tool_bar">
   <property name="windowTitle">
    <string>Toolbar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::NoToolBarArea</set>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_file_open"/>
   <addaction name="action_file_close"/>
   <addaction name="separator"/>
   <addaction name="action_edit_zoom_out"/>
   <addaction name="action_edit_zoom_in"/>
  </widget>
  <widget class="QMenuBar" name="menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_file_open"/>
    <addaction name="action_file_open_recent"/>
    <addaction name="separator"/>
    <addaction name="action_file_close"/>
    <addaction name="action_file_close_others"/>
    <addaction name="action_file_close_all"/>
    <addaction name="separator"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menu_edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_edit_zoom_in"/>
    <addaction name="action_edit_zoom_out"/>
    <addaction name="action_edit_zoom_reset"/>
    <addaction name="separator"/>
    <addaction name="action_edit_find"/>
    <addaction name="action_edit_select_all"/>
   </widget>
   <widget class="QMenu" name="menu_settings">
    <property name="title">
     <string>Se&amp;ttings</string>
    </property>
    <widget class="QMenu" name="menu_text">
     <property name="title">
      <string>&amp;Text</string>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/icons/text.png</normaloff>:/icons/text.png</iconset>
     </property>
     <addaction name="action_word_wrap"/>
     <addaction name="action_indent_xml"/>
     <addaction name="action_scroll_to_bottom"/>
    </widget>
    <widget class="QMenu" name="menu_images">
     <property name="title">
      <string>&amp;Images</string>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/icons/image.png</normaloff>:/icons/image.png</iconset>
     </property>
     <addaction name="action_transparent_background"/>
    </widget>
    <widget class="QMenu" name="menu_common">
     <property name="title">
      <string>&amp;Common</string>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/icons/common.png</normaloff>:/icons/common.png</iconset>
     </property>
     <addaction name="action_close_deleted_files"/>
    </widget>
    <addaction name="menu_images"/>
    <addaction name="menu_text"/>
    <addaction name="menu_common"/>
   </widget>
   <widget class="QMenu" name="menu_about">
    <property name="title">
     <string>&amp;About</string>
    </property>
    <addaction name="action_about_file_monitor"/>
   </widget>
   <widget class="QMenu" name="menu_session">
    <property name="title">
     <string>&amp;Session</string>
    </property>
    <addaction name="action_session_save_as"/>
    <addaction name="action_session_open"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_session"/>
   <addaction name="menu_edit"/>
   <addaction name="menu_settings"/>
   <addaction name="menu_about"/>
  </widget>
  <action name="action_file_open">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/file_open.png</normaloff>:/icons/file_open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="toolTip">
    <string>Open File</string>
   </property>
   <property name="statusTip">
    <string>Open a file into a new tab.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_file_close">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/file_close.png</normaloff>:/icons/file_close.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="toolTip">
    <string>Close File</string>
   </property>
   <property name="statusTip">
    <string>Close the current file.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="action_edit_zoom_in">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/file_zoom_in.png</normaloff>:/icons/file_zoom_in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom &amp;In</string>
   </property>
   <property name="toolTip">
    <string>Zoom In</string>
   </property>
   <property name="statusTip">
    <string>Zoom into current file.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="action_edit_zoom_out">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/file_zoom_out.png</normaloff>:/icons/file_zoom_out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom &amp;Out</string>
   </property>
   <property name="toolTip">
    <string>Zoom Out</string>
   </property>
   <property name="statusTip">
    <string>Zoom out of current file.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="action_about_file_monitor">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/about.png</normaloff>:/icons/about.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About File Monitor</string>
   </property>
   <property name="statusTip">
    <string>Learn more about File Monitor.</string>
   </property>
  </action>
  <action name="action_transparent_background">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Transparent Background</string>
   </property>
   <property name="toolTip">
    <string>Make Background Transparent</string>
   </property>
   <property name="statusTip">
    <string>Fill the transparent areas of images with a tile pattern.</string>
   </property>
  </action>
  <action name="action_word_wrap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Word Wrap</string>
   </property>
   <property name="toolTip">
    <string>Word wrap</string>
   </property>
   <property name="statusTip">
    <string>Wrap words within text files.</string>
   </property>
  </action>
  <action name="action_session_save_as">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/session_save_as.png</normaloff>:/icons/session_save_as.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save As</string>
   </property>
   <property name="iconText">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save Session</string>
   </property>
   <property name="statusTip">
    <string>Save the current set of open files (session) to a session file.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_session_open">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/session_open.png</normaloff>:/icons/session_open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="iconText">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open Session</string>
   </property>
   <property name="statusTip">
    <string>Open a previously saved set of open files (session) from a session file.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_close_deleted_files">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Close Deleted Files</string>
   </property>
   <property name="toolTip">
    <string>Close Deleted Files</string>
   </property>
   <property name="statusTip">
    <string>Automatically close a file's tab if it is deleted on disk.</string>
   </property>
  </action>
  <action name="action_indent_xml">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Format XML</string>
   </property>
   <property name="toolTip">
    <string>Format XML</string>
   </property>
   <property name="statusTip">
    <string>Cleanly indent and format XML files.</string>
   </property>
  </action>
  <action name="action_edit_zoom_reset">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/file_zoom_reset.png</normaloff>:/icons/file_zoom_reset.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Reset Zoom</string>
   </property>
   <property name="toolTip">
    <string>Reset Zoom</string>
   </property>
   <property name="statusTip">
    <string>Reset the zoom of the current file.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="action_file_open_recent">
   <property name="text">
    <string>Open &amp;Recent</string>
   </property>
   <property name="toolTip">
    <string>Open Recent</string>
   </property>
   <property name="statusTip">
    <string>Browse recently opened files.</string>
   </property>
  </action>
  <action name="action_file_close_others">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/file_close_others.png</normaloff>:/icons/file_close_others.png</iconset>
   </property>
   <property name="text">
    <string>Close O&amp;thers</string>
   </property>
   <property name="toolTip">
    <string>Close Others</string>
   </property>
   <property name="statusTip">
    <string>Close all files other than current file.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+W</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/file_quit.png</normaloff>:/icons/file_quit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit</string>
   </property>
   <property name="statusTip">
    <string>Close this application.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_scroll_to_bottom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Scroll To Bottom</string>
   </property>
   <property name="toolTip">
    <string>Scroll To Bottom</string>
   </property>
   <property name="statusTip">
    <string>Scroll to the bottom of text files (that are not XML) when they change on disk.</string>
   </property>
  </action>
  <action name="action_edit_select_all">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/edit_select_all.png</normaloff>:/icons/edit_select_all.png</iconset>
   </property>
   <property name="text">
    <string>Select &amp;All</string>
   </property>
   <property name="iconText">
    <string>Select All</string>
   </property>
   <property name="toolTip">
    <string>Select All Text</string>
   </property>
   <property name="statusTip">
    <string>Select all text in the current file.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_edit_find">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/edit_find.png</normaloff>:/icons/edit_find.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Find</string>
   </property>
   <property name="toolTip">
    <string>Find Text</string>
   </property>
   <property name="statusTip">
    <string>Find text within the current file.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_file_close_all">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/file_close_all.png</normaloff>:/icons/file_close_all.png</iconset>
   </property>
   <property name="text">
    <string>Close &amp;All</string>
   </property>
   <property name="statusTip">
    <string>Close all open files.</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
